boards: nucleo_wb55rg: Enable flash and debug with pyocd

erwango opened the pull request:
stm32wb is not yet supported on openocd.
But support on pyocd can be enabled thanks to "pack" feature.
Configure board runner with pyocd and provide guidelines
to configure pyocd.


Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder commented on boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst:
```diff
@@ -209,8 +209,13 @@ Flashing
-included in the Zephyr SDK. You can flash your application with drag and drop
-in the drive mounted when plugging your nucleo board to your PC.
+included in the Zephyr SDK.
+Though, support can be enabled on pyocd. It is not supported "builtin", but you
```
Better would be:
```
Instead, support can be enabled on pyocd by
adding "pack" support with the following pyocd command:
```

erwango replied:
@dbkinder, Fixed, txs!

dbkinder reviewed (approved):
doc changes LGTM, thanks!

nashif commented on boards/arm/nucleo_wb55rg/doc/nucleowb55rg.rst:
```diff
@@ -242,18 +246,11 @@ You should see the following message on the console:
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
+   :zephyr-app: samples/basic-blinky
```
basic-blinky? Maybe samples/basic/blinky

dbkinder replied:
yep, good catch.  The sample is in ``samples/basic/blinky``

erwango replied:
Txs, fixed.

erwango committed:
boards: nucleo_wb55rg: Enable flash and debug with pyocd

stm32wb is not yet supported on openocd.
But support on pyocd can be enabled thanks to "pack" feature.
Configure board runner with pyocd and provide guidelines
to configure pyocd.


Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>
